include/nccl_ofi_rocm.h

```diff
@@ -44,6 +44,43 @@ int nccl_net_ofi_cuda_get_num_devices(void);
  */
 int nccl_net_ofi_cuda_get_active_device_idx(void);
 
+/*
+ * @brief wraps hipMalloc()
+ * @return	0 on success
+ *		-1 on error
+ */
+int nccl_net_ofi_cuda_mem_alloc(void **ptr, size_t size);
```

Comment:
we already have some `_cuda_` names in this file so I'm fine with it in this patch, but at some point we should rename all these with the `_gpu_` convention.